draft-ietf-quic-invariants.md

```diff
@@ -252,8 +252,9 @@ Packets for the same QUIC connection might use different connection ID values.
 
 ## Version
 
-QUIC versions are identified with a 32-bit integer, encoded in network byte
-order.  Version 0 is reserved for version negotiation (see
+The Version field contains a 32-bit integer, encoded in network byte order.
+This value can be used by endpoints to identify a QUIC Version.  A Version field
```

Comment:
If we end up with version ossification and the real version ends up in transport params, is it still the case that an endpoint can use the Version field to identify a QUIC Version? Perhaps "This value can be used by endpoints in identifying a QUIC Version" instead. That way there's room that an endpoint might consider other factors in identifying the QUIC Version used.

Reply:
Ahh, this gets into whether it is necessary or sufficient information to identify a QUIC version and whether we need to say anything more.  "can" might be read to imply that it is sufficient, though "can" doesn't imply necessity.  But "can" also means that it is possible for endpoints to use the field.  That is the meaning I intended and I think that's OK.

Reply:
That wiggle room around "can" sounds good to me.